CityOfDetroit/bloom
# fix: allow deletion of users

# Pull Request Template

## Issue Overview

This PR addresses #1464


## Description

The delete users endpoint currently doesn't work in Detroit. This is due to trying to save an activity log with a null record id. It was fixed in core but never carried over. The main change that was needed was the delete endpoint needed to have the id be in the body instead of part of the url path.

With this change it also required me to update the application delete endpoint to match the same format. However, we don't have application functionality so this was just made to get the tests to work again.

## How Can This Be Tested/Reviewed?

You should be able to open a user and delete it if you have the correct privilege.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
